### PR TITLE
DSD-474: DS Select component with Chakra's Select component

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-474](https://jira.nypl.org/browse/DSD-474)

## This PR does the following:
- Updates the DS `Select` component to use Chakra's `Select` component.
- The deprecated examples were removed.

## How has this been tested?

Locally and through Storybook.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr683-4ptrkz1hten1qaiftxgelpvngxpsxdw1.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6144f273b8fde51bddb0adb2

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr683-4ptrkz1hten1qaiftxgelpvngxpsxdw1.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6144f273b8fde51bddb0adb2

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr683-4ptrkz1hten1qaiftxgelpvngxpsxdw1.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6144f273b8fde51bddb0adb2

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr683-4ptrkz1hten1qaiftxgelpvngxpsxdw1.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6144f273b8fde51bddb0adb2

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr683-4ptrkz1hten1qaiftxgelpvngxpsxdw1.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6144f273b8fde51bddb0adb2

File: src/components/Select/Select.stories.mdx [hunk 1]
```diff
@@ -39,198 +38,214 @@ import { getCategory } from "../../utils/componentCategories";
 | Component Version | DS Version |
 | ----------------- | ---------- |
 | Added             | `0.7.0`    |
-| Latest            | `0.24.0`   |
+| Latest            | `0.24.2`   |
```
Comment (bigfishdesign13) [inline src/components/Select/Select.stories.mdx]: Shoiudl be changed to `0.25.1`.

File: src/components/Select/Select.stories.mdx [hunk 2]
```diff
@@ -39,198 +38,214 @@ import { getCategory } from "../../utils/componentCategories";
...
-The `Select` component renders a `select` element along with its `option` children. For optimal accessibility, the `labelText` property is a required prop, regardless of the label visibility. Additionally, while the `id` prop is optional, a unique `id` attribute is necessary for accessibility. If the prop is left blank, a value will be generated for you.
+The `Select` component renders a `select` element along with its `option`
+children. For optimal accessibility, the `labelText` property is a required
+prop, regardless of the label visibility. Additionally, while the `id` prop is
+optional, a unique `id` attribute is necessary for accessibility. If the prop
+is left blank, a value will be generated for you.
 
-<Preview withToolbar>
+The `Select` component renders all the necessary wrapping components but the
+children _need_ to be `option` HTML elements.
```
Comment (bigfishdesign13) [inline src/components/Select/Select.stories.mdx]: I was not sure what this meant at first.  Can you adjust the language to something like this?
```
The `Select` component renders all the necessary wrapping and associated text elements, but the select options _need_ to be child `<option>` HTML elements.
```

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Looks good and works well.  Just a few minor changes requested.

**Comment by EdwinGuzman** [comment]
Updated.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr683-4ptrkz1hten1qaiftxgelpvngxpsxdw1.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6144f273b8fde51bddb0adb2

**Review by bigfishdesign13 — APPROVED** [review]
Looks good!

**Pull request merged**